dogfooding/src/main/kotlin/io/getstream/video/android/MainActivity.kt

```diff
@@ -66,6 +73,52 @@ class MainActivity : ComponentActivity() {
                     )
                 }
             }
+
+            checkForAppUpdates(viewModel.appUpdateResultFlow)
+        }
+    }
+
+    private suspend fun checkForAppUpdates(appUpdateResultFlow: Flow<AppUpdateResult>) {
```

Comment:
Can we move this somewhere into a different class / util object? Only to keep the MainActivity a bit more simple because this code is noise for external developers that want to see how to use our SDK and this code will confuse them.

Reply:
Good idea. I've created a a helper class and moved as much code as possible into it.